--- a/net/minecraft/data/Main.java
+++ b/net/minecraft/data/Main.java
@@ -67,7 +_,7 @@
 
 public class Main {
     @SuppressForbidden(reason = "System.out needed before bootstrap")
-    public static void main(final String[] args) throws IOException {
+    static void main(final String[] args) throws IOException {
         SharedConstants.tryDetectVersion();
         OptionParser parser = new OptionParser();
         OptionSpec<Void> helpOption = parser.accepts("help", "Show the help menu").forHelp();
