--- a/net/minecraft/client/renderer/blockentity/BeaconRenderer.java
+++ b/net/minecraft/client/renderer/blockentity/BeaconRenderer.java
@@ -106,16 +_,10 @@
         float texVOff = Mth.frac(scroll * 0.2F - Mth.floor(scroll * 0.1F));
         poseStack.pushPose();
         poseStack.mulPose(Axis.YP.rotationDegrees(animationTime * 2.25F - 45.0F));
-        float wnx = 0.0F;
-        float enz = 0.0F;
-        float wsx = -solidBeamRadius;
-        float wsz = 0.0F;
-        float esx = 0.0F;
+        final float wsx = -solidBeamRadius;
         float esz = -solidBeamRadius;
-        float uu1 = 0.0F;
-        float uu2 = 1.0F;
-        float vv2 = -1.0F + texVOff;
-        float vv1 = height * scale * (0.5F / solidBeamRadius) + vv2;
+        final float vv2 = -1.0F + texVOff;
+        final float vv1 = height * scale * (0.5F / solidBeamRadius) + vv2;
         submitNodeCollector.submitCustomGeometry(
             poseStack,
             RenderTypes.beaconBeam(beamLocation, false),
@@ -124,14 +_,12 @@
             )
         );
         poseStack.popPose();
-        wnx = -beamGlowRadius;
+        float wnx = -beamGlowRadius;
         float wnz = -beamGlowRadius;
-        enz = -beamGlowRadius;
-        wsx = -beamGlowRadius;
-        uu1 = 0.0F;
-        uu2 = 1.0F;
-        vv2 = -1.0F + texVOff;
-        vv1 = height * scale + vv2;
+        float enz = -beamGlowRadius;
+        float wsx_1 = -beamGlowRadius;
+        float vv2_1 = -1.0F + texVOff;
+        float vv1_1 = height * scale + vv2_1;
         submitNodeCollector.submitCustomGeometry(
             poseStack,
             RenderTypes.beaconBeam(beamLocation, true),
@@ -145,14 +_,14 @@
                 wnz,
                 beamGlowRadius,
                 enz,
-                wsx,
+                wsx_1,
                 beamGlowRadius,
                 beamGlowRadius,
                 beamGlowRadius,
                 0.0F,
                 1.0F,
-                vv1,
-                vv2
+                vv1_1,
+                vv2_1
             )
         );
         poseStack.popPose();
